tf-modules/cumulus/ecs_cluster.tf

```diff
@@ -192,21 +192,15 @@ resource "aws_security_group" "ecs_cluster_instance" {
   tags   = var.tags
 }
 
-resource "aws_security_group_rule" "ecs_cluster_instance_allow_ssh" {
-  type              = "ingress"
-  from_port         = 2
-  to_port           = 2
-  protocol          = "-1"
-  cidr_blocks       = ["0.0.0.0/0"]
+resource "aws_vpc_security_group_ingress_rule" "ecs_cluster_instance_allow_ssh" {
+  ip_protocol       = "-1"
+  cidr_ipv4 = cidrsubnet(data.aws_vpc.current.cidr_block, 4, 1)
```

Comment:
There are a few instances in the terraform docs that demonstrate getting the subnet address from a VPC. e.g. https://registry.terraform.io/providers/hashicorp/aws/3.74.1/docs/data-sources/vpc

That's what I'm asserting this does. This assertion should be verified 😄 

Rather than leaving this wide open, we're saying the CIDR block is restricted to the VPC.